test/acceptance/partner_profile.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-env mocha */
+import { setup, teardown } from './helpers'
+import { server as antigravity } from 'antigravity'
+
+describe('Partner profile page', () => {
+  let gravity, browser
+
+  before(async () => {
+    ({ gravity, browser } = await setup())
+    gravity.use(antigravity)
```

Comment:
These specs are ported from [existing specs](https://github.com/artsy/force/blob/master/mobile/apps/partner_profile/test/integration.coffee) which use Antigravity to mock responses. So for the sake of not rewriting these completely I'm mounting [Antigravity's express server](https://github.com/artsy/antigravity/blob/master/node/server.coffee) to provide some initial functionality.